themes/default/content/blog/pulumi-kubernetes-new-2022/index.md

```diff
@@ -0,0 +1,187 @@
+
+* [Pulumi Kubernetes Operator v1.10](https://github.com/pulumi/pulumi-kubernetes-operator/#readme): New integration with Flux for richer GitOps support and ability to deploy Pulumi stacks from directly within the Kubernetes resource model
+* [New Pulumi Provider for Flux](https://www.pulumi.com/registry/packages/flux/): Manage Flux with Infrastructure as Code
+* [Pulumi Kubernetes Provider v3.22](https://www.pulumi.com/registry/packages/kubernetes/): Server Side Apply and Resource Patch
+
+You can [learn more about Pulumi and Kubernetes](https://www.pulumi.com/registry/packages/kubernetes/) or jump in and [get started](https://www.pulumi.com/docs/get-started/kubernetes/) right away, or read on to learn more about what’s new for Pulumi and Kubernetes!
+
+## Pulumi Kubernetes Operator v1.10: Flux Integration and Inline Infrastructure support
+
+The Pulumi Kubernetes Operator provides a cloud-native way to manage and deploy cloud infrastructure using Pulumi from within your Kubernetes environment.  Infrastructure as Code defined in Pulumi can be automatically deployed using a GitOps model driven from within a Kubernetes environment - leveraging existing Kubernetes-based orchestration, RBAC, policy and resource model investments.  We released [Pulumi Kubernetes Operator 1.0](https://www.pulumi.com/blog/pulumi-kubernetes-operator-1-0/) last year, and since then have seen users scale up usage of the Operator to drive hundreds of thousands of Pulumi infrastructure updates.
+
+Pulumi Kubernetes Operator v1.10 expands the ways that Pulumi infrastructure can be defined.  In addition to the existing Git-based source that has been supported to date, this release adds two new ways to define your infrastructure:
+
+* Integration with Flux Sources - enabling any Flux Source to provide the Pulumi program that defines your cloud infrastructure.
+* Support for Inline Infrastructure sources - a new `pulumi.com/v1/Program` Custom Resource that can be used to describe cloud infrastructure directly within the Kubernetes resource model.
+
+Together, these open up a wide variety of new ways to apply the Kubernetes Operator as an interface for defining and delivering cloud infrastructure.
+
+### Integration with Flux Sources
+
+<img align="right" width="120" src="flux-icon.png">
+
+We’re excited to partner with the [Flux](https://fluxcd.io/) project in the CNCF to bring even richer GitOps support to the Pulumi Kubernetes Operator.  While the Pulumi Kubernetes Operator offers basic GitOps and branch tracking support, it supports only a fraction of the features that the Flux’s SourceController providers. Instead of rebuilding all of these features, we chose to provide rich integration with Flux Sources, enabling Pulumi programs to be provided via Flux, and then handed off to drive infrastructure deployments using the Pulumi Kubernetes Operator `Stack` resource.
```

Comment:
```suggestion
We’re excited to work with [Flux](https://fluxcd.io/), a CNCF project, to bring even richer GitOps support to the Pulumi Kubernetes Operator.  While the Pulumi Kubernetes Operator offers basic GitOps and branch tracking support, it supports only a fraction of the features that the Flux’s SourceController providers. Instead of rebuilding all of these features, we chose to provide rich integration with Flux Sources, enabling Pulumi programs to be provided via Flux, and then handed off to drive infrastructure deployments using the Pulumi Kubernetes Operator `Stack` resource.
```